src/api/auth/localStoarge.ts

```diff
@@ -0,0 +1,27 @@
+import type { LoginResponseDto } from "@/types/DTO/authDTO";
+
+const USER_INFO_KEY = "userInfo";
+const AUTH_TOKEN_KEY = "authToken";
```

Comment:
이렇게 하면 타입 추론을 string에서 리터럴 타입으로 더 좁힐 수 있겠네요!

```suggestion
const USER_INFO_KEY = "userInfo" as const;
const AUTH_TOKEN_KEY = "authToken" as const;
```